index.html

```diff
@@ -63,16 +61,13 @@
             promises.push(itowns.Fetcher.json('examples/layers/JSONLayers/WORLD_DTM.json').then(addLayerCb));
             promises.push(itowns.Fetcher.json('examples/layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addLayerCb));
 
-            menuGlobe.addGUI('RealisticLighting', false,
-                function(newValue) { globeView.setRealisticLightingOn(newValue); });
+            menuGlobe.addGUI('RealisticLighting', false, globeView.setRealisticLightingOn.bind(globeView));
 
-            Promise.all(promises).then(function () {
-                menuGlobe.addImageryLayersGUI(globeView.getLayers(function (l) { return l.type === 'color'; }));
-                menuGlobe.addElevationLayersGUI(globeView.getLayers(function(l) { return l.type === 'elevation'; }));
-                console.info('menuGlobe initialized');
-            }).catch( function (e) { console.error(e) });
+            // eslint-disable-next-line no-console
+            Promise.all(promises).then(menuGlobe.addLayersGUI.bind(menuGlobe)).catch(console.error);
 
-            globeView.addEventListener(itowns.GLOBE_VIEW_EVENTS.GLOBE_INITIALIZED, function () {
+            // eslint-disable-next-line prefer-arrow-callback
+            globeView.addEventListener(itowns.GLOBE_VIEW_EVENTS.GLOBE_INITIALIZED, function m() {
```

Comment:
I believe that this change isn't needed (linter rules are different for the examples)

Reply:
removed